bundles/org.openhab.binding.miio/README.md

```diff
@@ -806,22 +798,36 @@ e.g. `openhab:send actionCommand 'upd_timer["1498595904821", "on"]'` would enabl
 | HorizontalSwing      | Switch               | Fan-Horizontal Swing                     |            |
-| HorizontalAngle      | Number               | Fan-Horizontal Angle                     |            |
+| HorizontalAngle      | Number               | Fan-Horizontal Angle                     | Value mapping `["30"="30","60"="60","90"="90","120"="120","150"="150"]` |
 | Alarm                | Switch               | Fan-Alarm                                |            |
-| OffDelayTime         | Number               | Fan-Power Off Delay Time                 |            |
+| OffDelayTime         | Number:Time          | Fan - Power Off Delay Time               |            |
 | Brightness           | Switch               | Fan-Brightness                           |            |
 | MotorControl         | Number               | Fan-Motor Control                        |            |
 | SpeedLevel           | Number               | Fan-Speed Level                          |            |
 | PhysicalControlsLocked | Switch               | Physical Control Locked-Physical Control Locked |            |
 
+### Mi Smart Standing Fan Pro (<a name="dmaker-fan-p15">dmaker.fan.p15</a>) Channels
+
+| Channel              | Type                 | Description                              | Comment    |
+|----------------------|----------------------|------------------------------------------|------------|
+| actions              | String               | Actions                                  | Value mapping `["off-delay-time-toggle"="Off Delay Time Toggle"]` |
+| on                   | Switch               | Fan - Switch Status                      |            |
+| fan_level            | Number               | Fan - Gear Fan Level                     | Value mapping `["1"="Level1","2"="Level2","3"="Level3","4"="Level4"]` |
+| mode                 | Number               | Fan - Mode                               | Value mapping `["0"="Straight Wind","1"="Natural Wind"]` |
+| horizontal_swing     | Switch               | Fan - Horizontal Swing                   |            |
+| horizontal_angle     | Number               | Fan - Horizontal Angle                   | Value mapping `["30"="30","60"="60","90"="90","120"="120","140"="140"]` |
+| status               | Number               | Fan - Status                             |            |
+| on1                  | Switch               | Indicator Light - Switch Status          |            |
+| alarm                | Switch               | Alarm - Alarm                            |            |
+| fault                | Number               | Motor Controller - Device Fault          | Value mapping `["0"="No Faults"]` |
+| physical_controls_locked | Switch               | Physical Control Locked - Physical Control Locked |            |
+| off_delay_time       | Number               | Off Delay Time - Off Delay Time          |            |
```

Comment:
Can this be Number:Time? Is there any reason for using underscores instead of camelCase like the other Things?

Reply:
@codefalk can you maybe test / check in the app what is the unit for the delay time. I have no idea (e.g. seconds, minutes, hours?) e.g. by setting it in the app and see what is the response.
You can take the json file in this PR and save it in your misc/miio folder (see readme)

@fwolter the camelcase channels are made with the much older version of the conversion tooling, the newer version uses the underscore which is more in line with how Xiaomi is naming their channels.

Reply:
Thanks for taking this feature request up that quickly!

@marcelrv I checked as requested: "Off Delay Time" is in minutes. This works both ways, e.g. setting the parameter in OH as well setting the value via the Xiaomi Home App and read back in OH.

Reply:
Great I'll change the type then.
 Btw did all the other channels work fine ?

Reply:
okay, tested as much as I can:
all channels appear to work properly, However, few comments:
- I did not test the "action" channel (since I do not know which actions are valid ones)
- could not test the "fault" channel
- valid values for the "mode" channel are 0 for "direct breeze" and 1 for "natural breeze"
- valid values for the "horizontal_angle" channel are 30, 60, 90 120 and 140
- the "status" channel seems to be read only

Reply:
sorry, first time I deal with the enhanced miio binding feature and I just realized the above information is in the json file already.
For the action channel, "off-delay-time-toggle" just switches the fan off and on depending on it's current state. Not sure why the action is named "off-delay-time-toggle" since I can not identify any relation to "off_delay_time".

Reply:
no problem. I pushed the change to time, so I think we're good to go @fwolter